TestSwitchApi/TestSwitchApi.Tests/CandidatesListResponseTest.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+using TestSwitchApi.Controllers;
+using TestSwitchApi.Models.DataModels;
+using TestSwitchApi.Models.Request;
+using TestSwitchApi.Models.Response;
+using TestSwitchApi.Repositories;
+
+namespace TestSwitchApi.Tests
+{
+    public class CandidatesListResponseTest
+    {
+        public CandidatesListResponseTest()
+        {
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void TestingForNoCandidates()
+        {
+            var candidates = new List<CandidateDataModel>();
+            var pageRequest = new PageRequest
+            {
+                Page = 1,
+                PageSize = 10,
+            };
+            var candidatesListResponse = CandidateListResponse.Create(pageRequest, candidates, 0);
+            candidatesListResponse.TotalNumberOfItems.Should().Be(0);
+            candidatesListResponse.Items.Count().Should().Be(0);
+            candidatesListResponse.Page.Should().Be(1);
+            candidatesListResponse.PageSize.Should().Be(10);
+            candidatesListResponse.PreviousPage.Should().BeNullOrEmpty();
+            candidatesListResponse.NextPage.Should().BeNullOrEmpty();
+        }
+
+        [Test]
+        public void CheckingFirstPageOfCandidates()
+        {
+            var candidates = new List<CandidateDataModel>();
+            for (int cnt = 0; cnt < 10; cnt++)
```

Comment:
I'm not really sure what `cnt` stands for - is there a way to make it a bit more clear? Or you could just call it `i`.

Reply:
Have gone with i